### ISSUE TITLE
Error: 13 INTERNAL: Received RST_STREAM with code 2 triggered by internal client error: read ECONNRESET

### ISSUE THREAD

**Issue by htrappmaier** [googleapis/nodejs-datastore#812]
GCP transcoder job, started by firebase function on upload
=> `functions.storage.object().onFinalize(async (object) => {`

```
const request = {
       parent: transcoderServiceClient.locationPath(projectId, location),
       job: {
        inputUri: inputUri,
        outputUri: outputUri,
        config: jobconfig
       }
    };
     // Run request
     const [response] = await transcoderServiceClient.createJob(request);
```

**It only happens with a user not created via Expo RN in App, but created via JS in WebApp.**

  - OS: IOS
  - Node.js version: 12.18.3
  - npm version: 7.5.2
  - `@google-cloud/datastore` version: firebase 9.8.0

  1. Create User in webclient  vs. Create User in App
  2. start GCP Transcoder API job
  3. update reference doc in firestore (not sure, when the error occures)

thx,
Hermann

**Comment by meredithslota** [comment]
A fix for a possibly similar issue has been released, copied comment/info from here: https://github.com/googleapis/nodejs-firestore/issues/1023#issuecomment-1093017154 

> I promise it's not a belated April Fool's, but we actually have a fix: [grpc/grpc-node#2084](https://github.com/grpc/grpc-node/pull/2084) This will be part of the next `@grpc/grpc-js` release. 

It looks like the release that contained this fix is here: https://github.com/grpc/grpc-node/releases/tag/%40grpc%2Fgrpc-js%401.6.3 and just needs to propagate through and see if it also fixes this issue.

**Comment by meredithslota** [comment]
Since investigation and discussion were centralized in https://github.com/googleapis/nodejs-firestore/issues/1023 and we believe this issue to be fixed, please add additional repro info to that issue if this is still happening for you once you upgrade dependencies. Thanks!
